app/helpers/mobile_helper.rb

```diff
@@ -43,15 +43,9 @@ def show_comments_link(post, klass="")
       entypo_class = "entypo-chevron-down"
     end
 
-    if post.comments_count > 0
-      link_to "#{t('admins.stats.comments', count: post.comments_count)}<i class='#{entypo_class}'></i>".html_safe,
-              post_comments_path(post, format: "mobile"),
-              class: "show-comments #{klass}"
-    else
-      html = "<span class='show-comments'>"
-      html << t("admins.stats.comments", count: post.comments_count)
-      html << "</span>"
-    end
+    link_to "#{t('admins.stats.comments', count: post.comments_count)}<i class='#{entypo_class}'></i>".html_safe,
```

Comment:
Tagging a string as html safe may be a security risk, prefer `safe_join` or other Rails tag helpers instead.